libgloss/patmos/close.c

```diff
@@ -33,11 +33,20 @@
 #undef errno
 extern int  errno;
 
+//******************************************************************************
+/// patmos-plug: close: stub called if 'patmosplug_close(int)' is not defined.
+int _patmosplug_close(int file) {
+  errno = EBADF;
+  return -1;
+}
+
```

Comment:
Comment is needed for what this is used for. Perhaps:

```suggestion
/// patmosplug_clone: Alternative, patmos-specific `_close` implementation that can be provided
/// at program link time.
/// If not provided, will default to calling `_patmosplug_close`.
```